packages/flame/lib/src/components/base_component.dart

```diff
@@ -85,6 +86,10 @@ abstract class BaseComponent extends Component {
     }
   }
 
+  /// Runs after the component has been rendered, but before any children has
+  /// been rendered.
+  void postRender(Canvas canvas) {}
```

Comment:
```suggestion
  /// A life cycle callback that runs after the component has been rendered, 
  /// but before any children has been rendered.
  void postRender(Canvas canvas) {}
```

Just a doc suggestion giving a little bit of context on what this method is for.

Reply:
Is it a life cycle callback? Its a render cycle, life cycle is the whole game right? (onLoad and such)